decimal.go

```diff
@@ -900,12 +895,18 @@ func (d Decimal) LessThanOrEqual(d2 Decimal) bool {
 	return cmp == -1 || cmp == 0
 }
 
+// Betwee (LTE) return true when d is between d1 and d2 inclusive
```

Comment:
Go doc should start with the method name, so I'd suggest:
`IsBetween returns true when d is between (inclusive) d1 and d2, otherwise false.`